Commandant.xcodeproj/project.pbxproj

```diff
@@ -446,6 +482,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "org.carthage.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 3.0;
```

Comment:
This property seems to be generated automatically when sources had been converted.
I couldn't find this on GUI. So I guess we can't change it.